test/unit/entities/DutchQuote.test.ts

```diff
@@ -386,6 +387,41 @@ describe('DutchQuote', () => {
     });
   });
 
+  describe.only('fromResponseBody (RFQ)', () => {
+    process.env.ENABLE_PORTION = 'true';
```

Comment:
could prob remove these env statements, I think I needed them in order to isolate the test